### ISSUE TITLE
false values for EditProject don't get submitted

### ISSUE THREAD

**Issue by clofresh** [xanzy/go-gitlab#53]
Looks like when submitting boolean parameters, only the true parameters get sent, and the false ones get skipped.

```
p := &gitlab.EditProjectOptions{
	IssuesEnabled:        false,
	MergeRequestsEnabled: false,
	WikiEnabled:          true,
	SnippetsEnabled:      false,
	VisibilityLevel:      gitlab.InternalVisibility,
}
gl.Projects.EditProject(*project.ID, p)
```

I added some logging to the marshalling step:

```
&gitlab.EditProjectOptions{Name:"", Path:"", Description:"", DefaultBranch:"", IssuesEnabled:false, MergeRequestsEnabled:false, WikiEnabled:true, SnippetsEnabled:false, Public:false, VisibilityLevel:10}
"{\"wiki_enabled\":true,\"visibility_level\":10}"
```

I think it's hitting this issue: https://github.com/golang/go/issues/13284

**Comment by svanharmelen** [comment]
Thanks for the report @clofresh! My time to work on this package is limited at this moment, but I'll make sure I'll have a look at it...

As always a PR with a possible fix would be more then welcome :wink:

**Comment by svanharmelen** [comment]
Fixed by PR #66
